CRM/Core/BAO/Email.php

```diff
@@ -339,6 +339,49 @@ public static function getFromEmail() {
     return CRM_Utils_Array::crmArrayMerge($contactFromEmails, $fromEmailValues);
   }
 
+  /**
+   * Build "To Email" pulling data from Email entity
+   *
+   * @return array
+   *   an array of email
+   * @throws \CiviCRM_API3_Exception
+   */
+  public static function getToEmail() {
+    $contactID = CRM_Core_Session::singleton()->getLoggedInContactID();
```

Comment:
This bit is wrong - it's grabbing the contact for the currently logged in user, NOT the user we're trying to email to. This method should probably take an argument with the contact ID for who we want to email.